articles/manual/basics.md

```diff
@@ -44,15 +37,27 @@ In this way, Q# makes it easy to express the logic underlying quantum and hybrid
 
 ## Q# operations and functions
 
-Concretely, a Q# program is comprised of one or more *operations*, one or more *functions*, and user defined types. Operations are used to describe the transformations of the state of a quantum machine and are the most basic building block of Q# programs. 
-Each operation defined in Q# may then call any number of other operations, including the built-in *intrinsic* operations implemented by a target machine.
-When compiled, each operation is represented as a .NET class type that can be provided to target machines.
+ Concretely, a Q# program is comprised of one or more *operations*, one or more *functions*, and user defined types. Operations are used to describe the transformations of the state of a quantum machine and are the most basic building block of Q# programs. 
+ Each operation defined in Q# may then call any number of other operations, including the built-in *intrinsic* operations implemented by a target machine.
+ When compiled, each operation is represented as a .NET class type that can be provided to target machines.
```

Comment:
I recognize that this was there in the original, but it may be good to remove this sentence now that we support Python as a host language as well as .NET languages.

Reply:
Ok, I removed the .NET reference in 1b962068a4e558bb9f0642c812f73217e3ae814d